src/styles/blocks/stars.scss

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  height: $star-size();
+
+  &__star {
+    @include starSquare($star-size);
+    margin-right: 4px;
+    background: url(../images/star.svg) no-repeat center;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+@for $starsCount from 1 through 5 {
+  .stars--#{$starsCount} .stars__star:nth-child(-n + #{$starsCount}) {
+    background: url(../images/star-active.svg) center;
+  }
+}
```

You can put this in stars block. And you don't need to mention stars__star because you unnecessary increase specifity. In your code you have 2 classes and 1 pseudoclass.
```suggestion
.stars {
  @for $starsCount from 1 through 5 {
    .&--#{$starsCount} > :nth-child(-n + #{$starsCount}) {
      background: url(../images/star-active.svg) center;
    }
  }
}
```